### PR TITLE
Add docs about using instance storage

### LINKED ISSUE #457: Use of instance storage for Docker storage
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
> /kind bug
/kind feature

**What happened**:
Tarmak only supports EBS storage for Docker storage for the workers

**What you expected to happen**:
I think we should be able to let Docker use the instance storage as well when that is available. Instance storage is by nature faster (IOPS and latency). This would be good for the workers instance pool.

**How to reproduce it (as minimally and precisely as possible)**:
Bring up a cluster with instances for workers that support instance storage (eg C5d.large)

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration**:
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by MattiasGees**
**What this PR does / why we need it**:
Docs on how to use instance storage 
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #457 

**Special notes for your reviewer**:

**Release note**:

```release-note
Add docs section about using AWS instance storage
```

### CODE REVIEW

File: docs/user-guide.rst [hunk 1]
```diff
@@ -707,6 +707,41 @@ policy permissions. `Information on how to correctly set these permissions can
 be found here
 <https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy>`_.
 
+Instance storage
+~~~~~~~~~~~~~~~~
+
+Certain AWS instance types have support for `instance storage
+<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html>`_.
+It is possible to use instance storage as ultra fast storage for your Docker
```
Comment (charlieegan3) [inline docs/user-guide.rst]: How about this instead:

> It is possible to configure instance stor(e/age) for containers with Tarmak by specifying an instance type that has instance stor(e/age) capabilities (e.g ``c5d.xlarge``) in the ``size`` parameter of your instance pool.

File: docs/user-guide.rst [hunk 2]
```diff
@@ -707,6 +707,41 @@ policy permissions. `Information on how to correctly set these permissions can
 be found here
 <https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy>`_.
 
+Instance storage
```
[the remark this answers is not in the document]
Comment (MattiasGees) [reply docs/user-guide.rst]: Yeah apparently they renamed it. I changed it as well. The link to the docs still mentions `storage` :)

File: docs/user-guide.rst [hunk 3]
```diff
@@ -707,6 +707,41 @@ policy permissions. `Information on how to correctly set these permissions can
...
+~~~~~~~~~~~~~~~~
+
+Certain AWS instance types have support for `instance storage
+<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html>`_.
+It is possible to use instance storage as ultra fast storage for your Docker
+containers with Tarmak. This can be activated by defining a instance type
+(e.g. ``c5d.xlarge``) that has instance storage capabilities in the ``size``
+parameter of your instance pool. Also make sure you don't define any volumes
+with the name ``data``.
```
[the remark this answers is not in the document]
Comment (MattiasGees) [reply docs/user-guide.rst]: It had to be `Docker` instead of data. Otherwise it would use an EBS volume.

File: docs/user-guide.rst (hunk 1 above)
Comment (MattiasGees) [reply docs/user-guide.rst]: Changed

**Comment by charlieegan3** [comment]
@MattiasGees maybe this should have a release note?

Perhaps this?

> Add docs section about using AWS instance storage

**Pull request merged**